R/README.md

```diff
@@ -1,6 +1,16 @@
 # R on Spark
 
 SparkR is an R package that provides a light-weight frontend to use Spark from R.
+### Installing sparkR
+
+Libraries of sparkR need to be created in `$SPARK_HOME/R/lib`. This can be done by running the script `$SPARK_HOME/R/install-dev.sh`.
```

Comment:
I think this is needed as mentioned in running sparkR from R studio. The
script there tries to access the lib location which might not be present by
default in the sparkR folder if the wrong version of R is selected by
default.


*Regards,*
*Shubhanshu Mishra*
Research Assistant,
iSchool at University of Illinois at Urbana-Champaign
--------------------------------------------------
*Website:* http://shubhanshu.com
*LinkedIn Profile: *http://www.linkedin.com/in/shubhanshumishra

Blog <http://smexyyweby.wordpress.com/>  || Facebook
<http://www.facebook.com/shubhanshu.mishra>  ||  Twitter
<http://www.twitter.com/TheShubhanshu>  || LinkedIn
<http://www.linkedin.com/in/shubhanshumishra>

On Tue, Jan 19, 2016 at 10:03 PM, Felix Cheung <notifications@github.com>
wrote:

> In R/README.md
> <https://github.com/apache/spark/pull/10836#discussion_r50212459>:
>
> > @@ -1,6 +1,16 @@
> >  # R on Spark
> >
> >  SparkR is an R package that provides a light-weight frontend to use Spark from R.
> > +### Installing sparkR
> > +
> > +Libraries of sparkR need to be created in `$SPARK_HOME/R/lib`. This can be done by running the script `$SPARK_HOME/R/install-dev.sh`.
>
> To clarify, I'm referring the part about "sparkR need to be created in
> $SPARK_HOME/R/lib. This can be done by running the script
> $SPARK_HOME/R/install-dev.sh."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/10836/files#r50212459>.
>